app/src/main/res/layout/activity_game.xml

```diff
@@ -68,13 +68,32 @@
             android:visibility="gone"
             android:layout_marginTop="24dp" />
 
-        <Button
-            android:id="@+id/songGuessSubmitButton"
-            android:layout_width="wrap_content"
-            android:layout_height="wrap_content"
+        <LinearLayout
```

Comment:
Why did you add another linear layout ?

Reply:
We can remove it later as those two buttons will never be shown at the same time.